docs/examples/fa-examples.md

```diff
@@ -21,7 +29,10 @@ def read_user_input(my_automaton):
         print("")
 ```
 
-With this code in hand, let's define a DFA and see it in action:
+### Deterministic finite automaton (DFA)
+
+To use this function, let's first define a DFA.
+For a detailed definiton, see [this wikipedia article on DFAs][wikipedia-dfa].
```

Comment:
@eliotwrobson I suppose it can be a matter of personal preference, but I tend to always capitalize "Wikipedia":

```suggestion
For a detailed definiton, see [this Wikipedia article on DFAs][wikipedia-dfa].
```